src/components/ui/Tiles/tiles.scss

```diff
@@ -0,0 +1,115 @@
+@import "../../../styles/vendors/include-media";
+
+[data-store-tiles] {
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  grid-template-columns: repeat(2, 1fr);
+  column-gap: var(--grid-gap-0);
+  row-gap: var(--grid-gap-0);
+
+  @include media(">=tablet") {
+    grid-template-rows: repeat(1, 1fr);
+    grid-template-columns: repeat(4, 1fr);
+    column-gap: var(--grid-gap-1);
+  }
+}
+
+[data-store-tile] {
+  min-width: 9rem;
+  height: 24rem;
+  border-radius: var(--border-radius-default);
+
+  .product-card[data-store-card] {
+    max-width: 100%;
+    height: 100%;
+    padding: 0;
+
+    &[data-card-variant="horizontal"] .product-card__heading {
+      margin-right: 0;
+
+      @include media(">=tablet") {
+        width: 60%;
+        max-width: 60%;
+      }
+    }
+
+    [data-store-card-image] {
+      height: 60%;
+      overflow: hidden;
+      border-radius: var(--border-radius-default) var(--border-radius-default) 0 0;
+
+      [data-gatsby-image-wrapper] {
+        width: 100%;
+        height: 100%;
+      }
+
+      @include media(">=tablet") {
+        height: 70%;
+      }
+    }
+
+    [data-store-card-content] {
+      flex-direction: column;
+      align-items: flex-start;
+      height: 40%;
+      padding: var(--space-2);
+      border-bottom-right-radius: var(--border-radius-default);
+      border-bottom-left-radius: var(--border-radius-default);
+
+      @include media(">=tablet") {
+        flex-flow: row wrap;
+        height: 30%;
+        padding: var(--space-3);
+      }
+    }
+
+    .product-card__title {
+      max-height: 2.5rem;
+      overflow: hidden;
+    }
+  }
+
+  @include media(">=tablet") {
+    height: 28rem;
+  }
+}
+
+[data-store-tiles="expanded-first"], [data-store-tiles="expanded-first-two"] {
+  grid-template-rows: auto 1fr;
+}
+
+[data-store-tiles="expanded-first"] > [data-store-tile]:first-child,
+[data-store-tiles="expanded-first-two"] > [data-store-tile]:first-child,
+[data-store-tiles="expanded-first-two"] > [data-store-tile]:nth-child(2) {
+  grid-column: span 2;
+  min-width: 12rem;
+  height: fit-content;
+
+  [data-store-card-image] {
+    height: calc((100vw - 2rem) * .65);
```

Comment:
👀 How does it work? Is there any value that we can use as variable?

Reply:
hahahahah it's a fix for now, but we should start using aspect ratio on those images.

Reply:
It's a magic made by @renatamottam 🪄  hahahah